### ISSUE TITLE
Medical copayment IA/content strategy  from the Debt Resolution team

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#27780]
## What do you need content guidance and support for?

- [ ] Content strategy and writing to create new unauthenticated (static) VA.gov page
-	[x] Writing/editing to update existing unauthenticated (static) VA.gov page
-	[x] Content support for an app or tool
-	[ ] Something else (please add details in the next question)

## What do you need help with?

Per a conversation with @DanielleThierryUSDSVA, we'd like to use this ticket to track IA/content strategy for the medical copayment work so everyone's aligned on recommendations for H1s, URLs, and other content changes.

## When does this content work need to be done?

This tool is expected to launch in September 2021 

## Do you plan to bring this to an upcoming content office hours session?

Since this is a ticket for ongoing work, I plan to attend content office hours as needed!

- [x] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, chatbot content office hours (Fridays, 2:00 p.m. to 3:00 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: 
-	OCTO-DE product owner: Matthew Self
-	Product manager: @denisecoveyduc  
-	Designer: @rileyorr, @rebeccawalsh 
-	FE engineer: @craigwheeler, @LinnJS 
-	Product/team Slack channel: #vsa-debt

## Links to related tickets

- [Mobile UXPin prototype](https://preview.uxpin.com/361636c369f65453b4880d1445911c4d9b869349#/pages/140027243/simulate/no-panels)
- [Desktop UXPin prototype](https://preview.uxpin.com/18073b171729031ac0af64822d31a7427b012d66#/pages//simulate/sitemap?mode=i) (Linking because there's existing content feedback on that prototype - some has been consolidated and some comments have follow-up questions)
- [IA request ticket containing the financial hub's crosslinking diagram](https://github.com/department-of-veterans-affairs/va.gov-team/issues/25663#issuecomment-885048563)

**Comment by rileyorr** [comment]
Hello @bethpottsVADEPO & @DanielleThierryUSDSVA! During our usability study we tested a new 'online tool card' component ([View the component here](https://preview.uxpin.com/361636c369f65453b4880d1445911c4d9b869349#/pages//simulate/no-panels?mode=i)) to give Veterans more information about what they could expect to do within the tool before prompting them to sign in. The component tested well and folks understood what they could accomplish in the tool, however we were advised by the design council to stick with the existing sign in widget as it's customizable. 

I'm playing around with how we could draw attention to the tool's features before users log in, and I have three variations in UXPin:
- [Bullet points within the sign in alert](https://preview.uxpin.com/aaa18573afb5e4a4b9967a35be14696a20490811#/pages/140666253/simulate/no-panels?mode=i)
This is most similar to how we laid out the information in the online tool card. Participants generally scanned the text and understood what to expect with in the tool.
- [Full sentences within the sign in alert](https://preview.uxpin.com/aaa18573afb5e4a4b9967a35be14696a20490811#/pages/140666252/simulate/no-panels?mode=i)
I based this format off the sign-in alert component in the VA design system Sketch library - I couldn't find a similar example on staging
- [Additional section below the sign-in alert highlighting what can be accomplished within the tool](https://preview.uxpin.com/aaa18573afb5e4a4b9967a35be14696a20490811#/pages/140666251/simulate/no-panels?mode=i)
I've seen a lot of examples of this on the site including [the Claim Status Tool](https://staging.va.gov/claim-or-appeal-status/) and [VA lab and test results tool](https://staging.va.gov/health-care/view-test-and-lab-results/). I'm not sure that this pattern solves the problem, as we're asking Veterans to sign in before they see what the tool's about. 

I'm going to attend today's content office hours to chat about this - just wanted to post in here in case y'all wanted to look at the designs beforehand!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @rileyorr! @bethpottsVADEPO and I actually took this as a cross-site action item coming out of Friday's design council meeting. We're in the process of updating tool landing page content patterns, and are going to be looking at these alerts as part of that. Happy to talk about this more in office hours. Now that we're looking to add a tool to the pay your copay page, I'd like to also look at the overall page content and align it with what we're doing on other tool pages. I can share more about this when we chat. Thanks!

**Comment by rileyorr** [comment]
@DanielleThierryUSDSVA & @bethpottsVADEPO I incorporated the changes we talked about earlier! You can view the changes in [this UXPin prototype](https://preview.uxpin.com/aaa18573afb5e4a4b9967a35be14696a20490811#/pages/140666253/simulate/no-panels?mode=i). I've also attached a screenshot below. I think y'all mentioned you'd like some of the content to be adjusted as well - let me know what updates you'd recommend! Appreciate the collaboration~

![mcp-sign-in-widget.png](https://images.zenhubusercontent.com/5d66a8f72a515f013c440719/dc8c856b-931e-4927-93d0-48f37b3eba12)

**Comment by rebeccawalsh** [comment]
Hey Danielle! 

Quick feedback request on a small change we made! We had a participant who wasn't sure there were multiple balances to pay on the summary page (on mobile), so we did a couple of things to mediate that:

1. Shortened the cards vertically, **and**
2. Adjusting the heading above the cards from, "What you owe to each facility" to "What you owe to your [#] facilities"

Do you think #2 is an appropriate change, or are we breaking rules?

[UXPin link](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages//simulate/no-panels)

**Comment by rebeccawalsh** [comment]
## Are prescriptions included?

During our testing, we found that some participants weren't sure if both procedure/visit charges and prescription charges fell under the umbrella of 'copays' - they weren't sure if they'd find prescriptions in our tool. We did two things to mediate this:
- Improved the sign in alert on the content page to ensure prescription charges were specifically mentioned, **and**
- Changed the subtitle on the copayment summary page:
   - From: Check your VA health care copay balances. Find out how to make payments or request financial help." 
   - To: "Check your VA health care and prescription charges from each of your facilities. Find out how to make payments or request financial help."

[UXPin Prototype](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages//specification/no-panels)
[Design ticket](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/department-of-veterans-affairs/va.gov-team/27939)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @rileyorr and @rebeccawalsh. Did some additional thinking on the static content page and wanted to share with you. This is the page with revisions based on our sitewide content iterations on tool landing pages (now that this page is going to host a sign-in widget for a tool) + your iterations to the sign-in widget itself (with additional content tweaks).  Thoughts/questions? Let me know if you want to chat more.

[Pay your copay bill page_8.3.21](https://app.zenhub.com/files/133843125/17e79593-1c1e-4262-b9a0-1b36b0011f4f/download)

Beth and I are going to check in with Ryan re: the sign-in widget as well.

And then, since this static page lives in Drupal, the sitewide content team can implementation changes. But can you tell me the SMEs you're working with for the tool? We have SMEs that we work with for the page, but I want to make sure we're sharing changes with everyone who should see them. I also want to confirm a few terminology things with SMEs:
- Copay statement vs. billing statement vs. Veteran patient statement (refer to my comment in the attached doc)
- Copay vs. copayment (I believe SEO supports "copay" but in COVID-19-related content, SMEs have used "copayment")

I'll also look at the questions/changes you posted above as soon as I can.

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA! We have quite a list of SME's, and I believe Becca is currently mapping them out in Miro and will drop a link in this ticket. We did a round of SME interviews back in February and so many folks recommended we talk to David Adcock (david.adcock@va.gov) so he's our go-to SME at this point 😆 Mari-Beth Nye (maribeth.nye@va.gov) is another policy analyst who I believe works closely with David.  

> Copay statement vs. billing statement vs. Veteran patient statement (refer to my comment in the attached doc)

Veteran patient statement is the VHA term, however we learned through discovery research that Veterans think of these things as bills or billing statements. [Here's a link to that section of the readout](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/Debt%20Resolution/Medical_Copays/research/apr-2021/readout.md#how-do-veterans-conceptualize-the-mailed-veteran-patient-statements-as-debts-bills-something-else) so you can see what phrases folks used when we had folks look at a VPS and tell us what they'd call that document!

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA! Becca and I realized we've been posting a lot of one-off work in this ticket for feedback, but now we've made enough updates that it may be easier to just do a review of all the entire prototype ([which can be found here](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/140826057/specification/no-panels?mode=i)). I marked one page as 'in progress' because we aren't ready for a content review yet. 

Sorry for the spam in this ticket, but hopefully our comments about the UI updates will be helpful during review!

**Comment by LindaLi123** [comment]
@DanielleThierryUSDSVA  Hello Danielle, is there anything else we need to do before we can close this ticket?

**Comment by DanielleThierryUSDSVA** [comment]
Hi @LindaLi123. Yes, Riley and Becca had asked for a content review so we'll want to leave this ticket open until that's complete. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @rileyorr @rebeccawalsh! Questions as I'm going over more language things here:

- I know that when someone signs in and checks their balance on VA.gov, if they then want to pay online, we send them over to Pay.gov to do that. But I'd like to confirm a couple of things:
  - Does the person need to create a totally separate account to make a payment on Pay.gov? I see that you can sign in or create an account on Pay.gov, but I want to confirm.
  - They cannot check any balances or anything like that on Pay.gov, correct? It's just a form that allows them to submit a payment, right?

- Do we have any contacts with the folks who manage Pay.gov? If yes, can you send those names to me via slack or email? If not, I can see if I can find someone to connect with.

The reason I ask is because the language there is slightly different than ours. For example, they refer to the "Veteran patient statement" and I know we want to refer to "copay billing statement" (and I agree...that's why we named the existing page "Pay your VA copay bill" way back because SEO showed this language is how folks search for this info). It would be great if we could make sure there's consistent language throughout the experience to reduce confusion.

**Comment by DanielleThierryUSDSVA** [comment]
Adding some current SEO info here for our reference:

## Top organic search terms for va.gov/health-care/pay-copay-bill

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/b0c14add-968b-4785-8154-103a532d3aea)

## Top organic search terms for the full folder (including child pages)

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/f43fe75f-dafc-4135-8d55-80ef559f8484)

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/56d07b3c-a345-4fd2-aa55-d8f85e935958)

## Top organic search terms for https://www.pay.gov/public/form/start/25987221

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/ed615d09-bd51-4550-b0cf-12a8f38cdb94)

**Comment by rebeccawalsh** [comment]
Hey @DanielleThierryUSDSVA 

**Answers to your Q's:**
- No, they do not need to make an account for either pay.gov or pay.va.gov to make payments! I don't know the purpose of making an account. 
- No balances can be found online currently - they have to provide an account number from their mailed letters in order to pay.
- Finding people at pay.gov / pay.va.gov is in the works. [Thread about that here](https://dsva.slack.com/archives/CPE4AJ6Q0/p1628698597007400).

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA! 

Just wanted to give you a heads up about a couple adjustments we made to the content and things we recently learned:
- We added a section to the balance details page with instructions about disputing charges. I added a comment to that section of the UXPin, but I wasn't sure if you'd get an email notification about the comment. 
- We learned that we don't have access to any data attributes that correlate to 'balance statuses' (i.e. if the Veteran submits a FSR for a waiver they'd see a status that VHA has received their waiver request). This is a feature that we have in the debt portal.
   - I'm working on a more generalized blue status card with content about if the Veteran has requested financial assistance; I'll be attending this afternoon's content office hours to chat about which direction we should go in! 
   - You can [view the variations I have so far in this UXPin prototype](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/140983727/simulate/no-panels?mode=i)
- A VHA SME told us during a review of our wizard mapping terminals that in-person payments should be made at the VA medical facility the Veteran received the bill from because it can be cumbersome to get payments transferred to the correct facility if the Veteran pays at a different facility. 
   - This relates to content we have on [the 'Pay Your VA copay bill' content page](https://www.va.gov/health-care/pay-copay-bill/), as the in-person payment option directs Veterans to pay at their closest VA medical facility.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @rileyorr and @rebeccawalsh. Update on this:

## Static page

This page just isn't quite working framed as "Pay your VA copay bill" and then trying to add the tool that doesn't actually house the pay functionality at this time. So I've been re-strategizing the approach, and brainstorming with @mnorthuis, @bethpottsVADEPO, and @rtwell as well.

Here's a WIP doc of where this is at now. Since this is a Drupal page, the Sitewide Content team can manage this page going forward. Please let us know the estimated launch date for this tool.

### Additional question on the financial hardship page within the copay section

If the FSR will be incorporating copays in this iteration, we'll need to also make changes to the financial hardship static page. @RLHecht and I will connect with you on this when I'm back. 

## Tool

I added some feedback on content within the tool: https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/140669280/comment/no-panels?mode=i

We can't review all of the specific language in the tool in depth at this stage. So I would say to please iterate based on the feedback, and we can iterate on the wording details later (including post launch if needed). 

While I'm out next week, @bethpottsVADEPO is available during office hours for small questions as you go forward.

**Comment by DanielleThierryUSDSVA** [comment]
Attaching doc for static page.

[Pay your copay bill page with tool_08.12.21](https://app.zenhub.com/files/133843125/9d1489b8-28ad-4850-a467-5412e15f66a4/download)

**Comment by DanielleThierryUSDSVA** [comment]
@peggygannon wanted to keep you in the loop on this ticket. thanks!

**Comment by rileyorr** [comment]
@DanielleThierryUSDSVA thank you for the updates! I'll take a look at the static page and feedback on content within the tool. 

> Please let us know the estimated launch date for this tool.

@LindaLi123 do we have an estimated launch date for MCP? 

> If the FSR will be incorporating copays in this iteration, we'll need to also make changes to the financial hardship static page.

I believe this is on our roadmap as a fast follow-on feature after the MCP MVP launches. Linda or @leahkeeler may have a better picture of when we're looking to launch that feature so you and Randi can better estimate when we'll need to update that page.

> We can't review all of the specific language in the tool in depth at this stage. So I would say to please iterate based on the feedback, and we can iterate on the wording details later (including post launch if needed).

Sounds good to me! Would you like me to open a separate sitewide content intake request for that work so you and Randi can more easily track where it fits in with your workload?

**Comment by rileyorr** [comment]
@DanielleThierryUSDSVA I took a peek at the static content page doc you sent and added a couple comments that relate to the work we've been doing for the wizard. 

[Static content page with my comments](https://github.com/department-of-veterans-affairs/va.gov-team/files/6985092/9d1489b8-28ad-4850-a467-5412e15f66a4_RO.docx)

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @rileyorr! Very interesting about the phone number. I just called it to double check and it's the # for making electronic payments for your copay so I'm not sure what's changed. But definitely let us know what you find out. I wonder if they're wanting to switch it out for the # you call to get assistance?  That would be 866-400-1238. Do you want to ask the SME you're working with if that's the case?

**Comment by LindaLi123** [comment]
Hello @rileyorr the estimated launch date for MCP is around last week of September or first week of October.

**Comment by rileyorr** [comment]
Dropping this here so we can reference it later: 

- Ticket for 'notice of rights and responsibilities' content strategy assistance (#29489

**Comment by LindaLi123** [comment]
Hello @DanielleThierryUSDSVA is there anything we need to do for this ticket before we can close?

**Comment by DanielleThierryUSDSVA** [comment]
Hi @LindaLi123. We're working on this on our end (Mikki, Ryan, and I). I don't think there's anything for the debt resolution team to do on this specifically, other than confirming the phone #. But please leave the ticket open so we don't lose the thread. If having the ticket open is causing an issue for your team, let me know and I can create a separate ticket for our work and link to this for info. Thank you!

**Comment by LindaLi123** [comment]
Hello @DanielleThierryUSDSVA no problem we can leave this ticket open, just want to confirm all work from outside has been completed.

**Comment by rileyorr** [comment]
Hey @DanielleThierryUSDSVA! Due to data constraints, we're removing the table containing charges and payments from the MVP. Veterans will be able to download their monthly patient statements and learn about their resolution options. Just wanted to give you a heads up because the sign in widget on [our mock content page](https://preview.uxpin.com/16e8675a9898914d5615d6ea2d23e310b7d8dfaa#/pages/141834694/simulate/sitemap?mode=i) mentions that Veterans would be able to view their charges and payments (I have since removed that bullet point).

**Comment by DanielleThierryUSDSVA** [comment]
Okay, great - thanks, @rileyorr! I'll remove that from the landing page Mikki, Ryan, and I are working on as well.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @denisecoveyduc. Here's the content ticket. And here's the word doc with the content updates to the landing page that we've been working on. [Copay page revisions](https://app.zenhub.com/files/133843125/d258d4d4-8085-4de2-81f4-a3e5dc27a1e2/download)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @denisecoveyduc @rileyorr @LinnJS! Here's a [preview ](http://preview-prod.vfs.va.gov/preview?nodeId=704)of how we're thinking of restructuring the landing page to work in the new tool and streamline the page a bit while we're at it. As we talked about, however, the H2 for the tool sign in widget will appear with no widget until you launch to 100% of users. I'm thinking that what might be easiest is if you just use the current page for now and add that H2 above the sign-in component within the widget. Then, when you're ready to do the full launch, we can launch this version of the page. Let me know your thoughts - thanks!

**Comment by DanielleThierryUSDSVA** [comment]
Hi @RLHecht. The page at http://preview-prod.vfs.va.gov/preview?nodeId=704 is ready for copy editing. We'll want to hold on publishing, as I'll first take this to our VHA partners for awareness and because we'll need to wait until the tool is launched to 100% of users.

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA I copyedited this and it's ready to go when the tool is launched.

**Comment by DanielleThierryUSDSVA** [comment]
Thank you, @RLHecht!

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA! I finally got a chance to take a look at the copay page revisions. Thanks for sending that over! 

Regarding your comment about providing something around how Veterans can find their account number(s), we're going to display the account number in each facility's details page so folks won't need to download their statements to find that info. Just wanted to throw that out there in case you wanted to incorporate that into the instructions on how to find that information. 

Also, I can't access the preview link to view how y'all restructured the page. 😓

**Comment by rileyorr** [comment]
Hi @DanielleThierryUSDSVA and @RLHecht! We're waiting for final approval to launch, hoping for today but tomorrow at the latest. Should I open a separate content request ticket for the restructured landing page to be published?

**Comment by RLHecht** [comment]
Exciting @rileyorr! I can create a ticket for our team for publishing when you give us the go-ahead. I'll close out this ticket once I open the new one.

**Comment by denisecoveyduc** [comment]
Woo Hoo!  @RLHecht We are currently planning to go live tomorrow morning (Jan 13th at 9am EST) 🎉

**Comment by RLHecht** [comment]
Now that the tool has launched just confirming we can close out this ticket.

**Comment by denisecoveyduc** [comment]
@RLHecht Unless @rileyorr sees any reason that I may be missing I think we are ok to close this now that we have launched. I know (via Slack) @DanielleThierryUSDSVA  is already working on content changes to establish more equal framing on the "Manage Debt" page, which I assume we should capture/track in a separate issue?

**Comment by rileyorr** [comment]
@RLHecht & @denisecoveyduc I don't see any reason to keep this ticket open! I'll likely open a second ticket soon as we move toward the combined VBA + VHA experience :)

**Comment by RLHecht** [comment]
Sounds good! Closing this ticket.
